core/invoker/build.gradle

```diff
@@ -32,17 +32,16 @@ ext.coverageDirs = [
     "${buildDir}/classes/scala/scoverage",
     "${project(':common:scala').buildDir.absolutePath}/classes/scala/scoverage"
 ]
-distDockerCoverage.dependsOn ':common:scala:scoverageClasses', 'scoverageClasses'
```

Comment:
I'm currently looking for a way to recover this deleted line. gradle6 doesn't work because of this line.

There is a compatibility issue where compiled classes are deleted after running the distDockerCoverage task. (It is difficult to find the cause.)

This line is related to this change: https://github.com/apache/openwhisk/pull/3685
It doesn't seem like a common usage. I'm spending quite a bit of time fixing this.

Reply:
Can we just migrate to the way that is used with Gradle6 rather than fixing the issue as is?

Reply:
@style95 It's solved by upgrading the version of the scoverage gradle plugin.